src/cluster/slot_migrate.h

```diff
@@ -55,8 +55,9 @@ enum class SlotMigrationStage { kNone, kStart, kSnapshot, kWAL, kSuccess, kFaile
 enum class KeyMigrationResult { kMigrated, kExpired, kUnderlyingStructEmpty };
 
 struct SlotMigrationJob {
-  SlotMigrationJob(int slot_id, std::string dst_ip, int dst_port, int speed, int pipeline_size, int seq_gap)
-      : slot_id(static_cast<int16_t>(slot_id)),
+  SlotMigrationJob(const SlotRange &slot_range, std::string dst_ip, int dst_port, int speed, int pipeline_size,
+                   int seq_gap)
+      : slot_range(slot_range),
```

Comment:
this is a bit weird so maybe rename the argument to `slow_range_in`?

Reply:
I don't quite understand the meaning of `slow_range_in`.

Reply:
`slot_range(slot_range)` is a bit weird since input arg and member shares the same naming. I think rename input argument to another name is a good way for this problem

Reply:
ok, does `slow_range_in` here mean there is `max_speed`? I wanted to confirm if you actually meant to express `slot_range_in` but made a typo. Because I didn’t react right away.